src/drivers/bitbucket_cloud.js

```diff
@@ -145,9 +140,33 @@ class BitbucketCloud {
       body
     });
 
+    if (autoMerge)
+      await this.prAutoMerge({ pullRequestId: id, mergeMode: autoMerge });
     return href;
   }
 
+  async prAutoMerge({ pullRequestId, mergeMode, mergeMessage = undefined }) {
+    winston.warn(
+      'Auto-merge is unsupported by Bitbucket Cloud; see https://jira.atlassian.com/browse/BCLOUD-14286. Trying to merge immediately...'
+    );
+    const { projectPath } = this;
+    const endpoint = `/repositories/${projectPath}/pullrequests/${pullRequestId}/merge`;
+    const body = JSON.stringify({
+      merge_strategy: {
+        merge: 'merge_commit',
+        rebase: 'fast_forward',
+        squash: 'squash'
+      }[mergeMode],
+      close_source_branch: true,
+      message: mergeMessage
+    });
```

Comment:
I do not fully follow this. Is `[mergeMode]` correctly applied here?

Reply:
> Yes, that's functionally correct, although a bit hacky:
> 
> ```js
> > const mergeMode = 'rebase'
> > {merge: 'merge_commit', rebase: 'fast_forward', squash: 'squash'}[mergeMode] == 'fast_forward'
> true
> ```

where are you testing this?

Reply:
> _where are you testing this?_

Node.js v17.8.0, although it may work as well on Internet Explorer 6. 😈

Reply:
I do not fully understand what you are trying to do.
Could you please explain it to me?

> it may work as well on Internet Explorer 6.

It does not

Reply:
ah! got it.

Reply:
It looks like you're faster with your brain that I am with my keyboard. ⚡

Reply:
We're just “translating” our common merge mode names to Bitbucket's weird ones.

Reply:
Resolved with https://github.com/iterative/cml/pull/932/commits/9b4dd9d285c62c65e81842aec881a825666df1e7